src/test/java/com/qalabs/seleniumbasics/spotify/e2e/SignUpPageTest.java

```diff
@@ -0,0 +1,65 @@
+package com.qalabs.seleniumbasics.spotify.e2e;
+
+import com.qalabs.javabasics.spotify.pages.SpotifyHomePage;
+import com.qalabs.javabasics.spotify.pages.SpotifySignUpPage;
+import com.qalabs.seleniumbasics.spotify.BaseTest;
+import com.qalabs.seleniumbasics.spotify.utilities.PropertyReader;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import utils.Log;
+
+import java.util.List;
+
+public class SignUpPageTest extends BaseTest{
+    private String spotifyUrl;
+
+    @BeforeMethod
+    public void initSetup(){
+        PropertyReader propertyReader= new PropertyReader();
+        spotifyUrl= propertyReader.getProperty("credentials.properties", "URL_WEBSITE");
+        driver.navigate().to(spotifyUrl);
+        SpotifyHomePage spotifyHomePage = new SpotifyHomePage(driver);
+        Assert.assertTrue(spotifyHomePage.isLoaded(), "Página principal no se cargo correctamente");
+        spotifyHomePage.goToSignUpPage();
+        SpotifySignUpPage spotifySignUpPage = new SpotifySignUpPage(driver);
+        Assert.assertTrue( spotifySignUpPage.isLoaded(), "Página de Login no se cargo correctamente" );
+    }
+
+    @Test(alwaysRun = true,description = "TC_US4_001/ Llenar el formulario para crear cuenta nueva")
+    public void spotifySignNoGender(){
+        PropertyReader propertyReader = new PropertyReader();
+        String email=propertyReader.getProperty("credentials.properties", "TC_US_04_001_EMAIL");
```

Comment:
Why are you repeating the same variables for each test method?

Create a list of user values where you can store the values from the credentials.properties file and call the list (myAwesomeList.get(0)) in the methods who receive the data.